draft-ietf-httpbis-http2bis.xml

```diff
@@ -5057,15 +5057,11 @@ cookie: e=f
         <li>TLS_PSK_WITH_AES_128_CCM_8</li>
         <li>TLS_PSK_WITH_AES_256_CCM_8</li>
       </ul>
-      <aside>
-        <t>Note:
-          This list was assembled from the set of registered TLS cipher suites at the time of
-          writing.  This list includes those cipher suites that do not offer an ephemeral key
-          exchange and those that are based on the TLS null, stream, or block cipher type (as
-          defined in <xref target="TLS12" section="6.2.3"/>).  Additional cipher suites with
-          these properties could be defined; these would not be explicitly prohibited.
-        </t>
-      </aside>
+      <aside><t>Note: This list was assembled from the set of registered TLS cipher suites when
+      <xref target="RFC7540"/> was developed.  This list includes those cipher suites that do not
+      offer an ephemeral key exchange and those that are based on the TLS null, stream, or block
```

Comment:
And should this say “TLS null stream or block”?

Reply:
"null" (that is, identity or `f : x -> x`) is a type of "cipher" that some suites offer. Scare quotes on "cipher" are necessary because it's exactly as bad as it sounds.

"stream", and "block" are also types of cipher.  RC4 is an example of the former, AES CBC modes the latter.

Reply:
Wow this text is unclear. What it’s saying is that AEAD is a kind of cipher distinct from stream, block, and null? Ok I guess.